### PR TITLE
getting-started: update code block to prompt

### PR DESCRIPTION
**Pull request by munoz0raul**
- Changed all code blocks to prompt to help customers understand when
    it is a host command and when it is a device command.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 725b77de4f56de0ddf4e94f9bc69a11096c29ef7 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/845/docs/artifacts/html/index.html

File: source/getting-started/flash-device/index.rst [hunk 1]
```diff
@@ -301,17 +310,21 @@ If the above methods to connect your device to the network don't work, try one
 of the following.
 
 - Temporarily enable and connect to the UART (see directions above in
-  the WiFi section) and determine available IP addresses with::
+  the WiFi section) and determine available IP addresses with:
 
-    # Ethernet
-    ip addr show eth0 scope global
+  .. prompt:: text
```
Comment (mike-scott) [inline source/getting-started/flash-device/index.rst:315]: This one didn't turn out the same as the others

Comment (munoz0raul) [reply source/getting-started/flash-device/index.rst:315]: This is a text file, not a bash script.

Comment (munoz0raul) [reply source/getting-started/flash-device/index.rst:315]: Sorry, you are right!

Comment (munoz0raul) [reply source/getting-started/flash-device/index.rst]: Sorry, I thought that it was like a file reference. Just fixed it.

**Comment by doanac** [comment]
Docs for da6c7055e6f084e1ff756e429d47d02d20d920ef are browesable at: https://ci.foundries.io/projects/fio-docs/builds/846/docs/artifacts/html/index.html

**Review by angolini — APPROVED** [review]
Great!

**Comment by doanac** [comment]
Docs for 49896181886f2030b55ab7fc9d5fc11f66a78149 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/847/docs/artifacts/html/index.html

**Review by MatthewCroughan — APPROVED** [review]
Nice commit, some good fixes in addition to your additions.

**Pull request merged**